addons/mail/static/tests/helpers/webclient_setup.js

```diff
@@ -90,7 +90,7 @@ function setupMessagingServiceRegistries({
         start() {
             const originalService = busService.start(...arguments);
             Object.assign(originalService, {
-                _beep() {}, // Do nothing
+                _poll() {}, // Do nothing
```

Comment:
why a rename here?

Reply:
Its not a rename but a removal, the `_beep` method is no longer part of the bus service. The diff is not clear indeed..

Reply:
_poll was added